pyproject.toml

```diff
@@ -69,8 +69,8 @@ doc = [
   "jupyterlite-sphinx",
   "sphinxcontrib-youtube<1.4",
   "sphinx-favicon>=1.0.1",
-  # Install nbsphinx in case we want to test it locally even though we can't load
-  # it at the same time as MyST-NB.
+  "ipykernel",
+  "pypandoc_binary",
```

Comment:
This causes the build to fail on my Macbook:

```
ERROR: Could not find a version that satisfies the requirement pypandoc_binary; extra == "doc" (from pydata-sphinx-theme[doc]) (from versions: none)
ERROR: No matching distribution found for pypandoc_binary; extra == "doc"
```

[Full log](https://gist.github.com/gabalafou/5adee73557fc08a9639a4c172b4282df)

Since you added a note in the docs to install Pandoc before trying to install the theme, I think we should probably remove `pypandoc_binary` from this list of dependencies, right? 

```suggestion
```